test/apps/iodemo/io_demo.cc

```diff
@@ -2391,6 +2423,23 @@ class DemoClient : public P2pDemoCommon {
                 } else {
                     bytes_completed = total_bytes_completed;
                     num_completed   = total_completed;
+
+                    if (opts().per_conn_info) {
+                        long iops  = (long)(num_completed / elapsed);
+
+                        conn_log << "total iops: " << iops;
+
+                        if (opts().window_size == 1) {
```

Comment:
can we unite common code with 2401-2408?
possibly by removing latency altogether, or if not - counting latency per operation

Reply:
did you mean counting latency for read and latency for write?

Reply:
yes, counting them separately